nx_gzip/nx_gzip.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2020 IBM
+# Author: Kalpana Shetty <kalshett@in.ibm.com>
+
+import os
+import shutil
+from avocado import Test, skipIf
+from avocado.utils import build, process, distro, git, archive, memory
+from avocado.utils.software_manager import SoftwareManager
+from avocado.utils.partition import Partition
+
+IS_POWER_NXGZIP = 'PowerNV' in 'ls /dev/crypto/nx-gzip'
```

Comment:
This check is improper. ls wont get executed here and hence will always be false.